docs_source/static.config.js

```diff
@@ -1,10 +1,79 @@
-import axios from 'axios'
+import React, { Component } from 'react';
+import axios from 'axios';
 
 export default {
   basePath: 'wizardo',
   getSiteData: () => ({
-    title: 'Wizardo',
+    siteTitle: 'Wizardo',
+    metaDescription: 'Wizardo is tool that allows you to create code generators to bootstrap new projects or new features in existing ones.',
+    favicon: 'https://s3-us-west-2.amazonaws.com/assets-general-im/wizardo/wizardo.ico',
+    logo: "https://s3-us-west-2.amazonaws.com/assets-general-im/wizardo/logo.png",
```

Comment:
`siteTitle`, `metaDescription` and `favicon` are using single quotes but `logo` and `urlWizardo` are using double quotes.

Make them consistent.